contrib/set-mesh-repo/armbian-armbian.list

```diff
@@ -0,0 +1 @@
+deb http://__PREFIX__.mirror.tomesh.net/armbian/apt/ __CODENAME__ main __CODENAME__-utils __CODENAME__-desktop
```

Comment:
I think we should use `__PREFIX__.mirror.debian.tomesh.net` or something similar, otherwise when we change this eventually to reclaim the `mirror.tomesh.net` we'd end break existing installs.